### ISSUE TITLE
Training Loss is Puzzling

### ISSUE THREAD

**Issue by XYZ-qiyh** [xy-guo/MVSNet_pytorch#11]
![image](https://user-images.githubusercontent.com/45621782/71957815-ae725d80-3229-11ea-8125-cf1193c75ccd.png)
I have re-trained the network using the same cofiguration, but the training loss don't converge.

**Comment by XYZ-qiyh** [comment]
![image](https://user-images.githubusercontent.com/45621782/72033737-1a0e0680-32ce-11ea-9a1a-64c3476aadcd.png)
Hello @xy-guo . Thanks for your amazing work. I have re-trained the network using the same cofiguration, but the training loss don't converge.
I don't think the training loss is very plausible. Do you have this problem??
Regards.

**Comment by kwea123** [comment]
hi, did you continue training? What does the final loss curve look like? Also does the depth in the image tab seem improving? My training loss also fluctuates a lot.

**Comment by zhiwenfan** [comment]
> ![image](https://user-images.githubusercontent.com/45621782/72033737-1a0e0680-32ce-11ea-9a1a-64c3476aadcd.png)
> Hello @xy-guo . Thanks for your amazing work. I have re-trained the network using the same cofiguration, but the training loss don't converge.
> I don't think the training loss is very plausible. Do you have this problem??
> Regards.

Have you ever try to use larger batchsize?

**Comment by xy-guo** [comment]
I use 8 gpus to train the model. It seems your training loss is converging, try using larger smoothing weight in tensorboard.

**Comment by kwea123** [comment]
Do you have the final metrics of `abs_depth_error`, `thres2mm_error`, `thres4mm_error`, `thres8mm_error`? I want to compare with my results, thank you.

**Comment by kwea123** [comment]
@xy-guo

**Comment by xy-guo** [comment]
Sorry I have lost the final metrics information. @kwea123

**Comment by whubaichuan** [comment]
@kwea123 Have you try bigger batchsize? What's the difference with the training loss of batchsize=1?

**Comment by kwea123** [comment]
The model consumes too much memory so I cannot try bigger batchsize (batchsize=1 requires ~8GB). You will need multiple gpus.

**Comment by whubaichuan** [comment]
@Qtodd Hi, have you solve the problem that the training loss is not plausible?

**Comment by whubaichuan** [comment]
@xy-guo Do you use 8 GPUS to test?

**Comment by kwea123** [comment]
I tried, even 8GPUs with batchsize 4 results still in fluctuating losses. I think it's insolvable then; although it doesn't worsen the model performance..
